partner_priority/models/partner_priority.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Patrick Wilson <patrickraymondwilson@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class PartnerPriority(models.Model):
+    _name = 'partner.priority'
+    _order = 'sequence'
+    _description = 'Partner Priority'
+
+    name = fields.Char(string="Priority", required=True)
+    description = fields.Text(string="Description")

```

Comment:
You do not need the strings if the are the same as the field name (they will automatically start with a capital).